stdlib/date/truncate_duration_test.flux

```diff
@@ -33,12 +45,38 @@ outData =
 ,,0,FF,_m,2018-05-22T19:53:40.000000000Z,1893459600000000000
 ,,0,FF,_m,2018-05-22T19:53:50.000000000Z,1893463200000000000
 ,,0,FF,_m,2018-05-22T19:54:00.000000000Z,1893466800000000000
-"
-t_duration_truncate = (table=<-) =>
-    table
-        |> range(start: 2018-05-22T19:53:00Z)
-        |> drop(columns: ["_start", "_stop"])
-        |> map(fn: (r) => ({r with _value: int(v: date.truncate(t: duration(v: r._value), unit: 1s))}))
-
-test _duration_truncate = () =>
-    ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_duration_truncate})
+",
+            )
+
+        testing.diff(got: got, want: want)
+    }
+
+testcase truncate_duration_location {
+        option location = timezone.location(name: "Europe/Madrid")
+
+        got =
+            csv.from(csv: inData)
+                |> range(start: 2018-05-22T19:53:00Z)
+                |> drop(columns: ["_start", "_stop"])
+                |> map(fn: (r) => ({r with _value: int(v: date.truncate(t: duration(v: r._value), unit: 1mo))}))
```

Comment:
The existing test also used `int(v: ...)` here which is why I assume you did but would it not be easier to read the expected output if we omitted the integer conversion?

Reply:
I tired to keep it consistent but I can just use a timestamp. I will change it.